docs/en/observability/logs-parse-filter.asciidoc

```diff
@@ -0,0 +1,1227 @@
+
+After creating your pipeline, an index template points your log data to your pipeline. Use the index template you created in the <<logs-stream-index-template, Extract the `@timestamp` field>> section.
+
+[discrete]
+[[logs-stream-reroute-add-logs]]
+== Add logs to your data stream
+
+Add the example logs to your data stream with this command:
+
+[source,console]
+----
+POST logs-example-default/_bulk
+{ "create": {} }
+{ "message": "2023-08-08T13:45:12.123Z WARN 192.168.1.101 Disk usage exceeds 90%." }
+{ "create": {} }
+{ "message": "2023-08-08T13:45:14.003Z ERROR 192.168.1.103 Database connection failed." }
+{ "create": {} }
+{ "message": "2023-08-08T13:45:15.004Z DEBUG 192.168.1.104 Debugging connection issue." }
+{ "create": {} }
+{ "message": "2023-08-08T13:45:16.005Z INFO 192.168.1.102 User changed profile picture." }
+----
+
+[discrete]
+[[logs-stream-reroute-verify]]
+== Verify the reroute processor worked
+
+The reroute processor should route any logs with a `log.level` of `WARN` or `ERROR` to the `logs-critical-default` data stream. Query the the data stream using the following command to verify the log data was routed as intended:
+
+[source,console]
+----
+GET log-critical-default/_search
+----
+
+Your should see similar results to the following showing that the high-severity logs are now in the `critical` dataset:
+
+[source,JSON]
+----
+{
+  ...
+  "hits": {
+    ...
+    "hits": [
+        ...
+        "_source": {
+          "host": {
+            "ip": "192.168.1.101"
+          },
+          "@timestamp": "2023-08-08T13:45:12.123Z",
+          "message": "Disk usage exceeds 90%.",
+          "log": {
+            "level": "WARN"
+          },
+          "data_stream": {
+            "namespace": "default",
+            "type": "logs",
+            "dataset": "critical"
+          },
+          {
+        ...
+        "_source": {
+          "host": {
+            "ip": "192.168.1.103"
+           },
+          "@timestamp": "2023-08-08T13:45:14.003Z",
+          "message": "Database connection failed.",
+          "log": {
+            "level": "ERROR"
+          },
+          "data_stream": {
+            "namespace": "default",
+            "type": "logs",
+            "dataset": "critical"
+          }
+        }
+      }
+    ]
+  }
+}
+----
+
+[discrete]
+[[logs-filter-and-aggregate]]
+= Filter and aggregate your logs
+
+After sending log data to {es} and <<logs-stream-parse, extracting structured fields>>, filtering and aggregating your data helps you find specific information, gain insight, and monitor your systems more efficiently. 
+
+* <<logs-filter>> — Narrow down your log data by applying specific criteria.
+* <<logs-aggregate>> — Analyze and summarize data to find patterns and gain insight.
+
+[discrete]
+[[logs-filter]]
+== Filter your logs
+
+Filter your data to focus on logs that meet specific criteria, such as log levels, timestamp ranges, or host IPs. You can filter your log data in different ways:
```

Comment:
In most cases, I hope users go to Log Explorer to do the work. Can we phrase this in a way that we say: We explain you here what is happening under the hood but Logs Explorer is what we recommend?

Reply:
If we want to focus on discover, maybe we don't need to show the inner workings here. We already covered some of querying and filtering with QDSL through the examples in the extracting fields sections, maybe at this point we talk specifically about logs Explorer. WDYT?

Reply:
I like that we show both in parallel. My only point is that we use Logs Explorer instead of Discover. But if you feel some examples are overkill and it can be achived much easier in Logs Explorer (what I hope is the case), happy to remove the samples. It will be interesting to see what the workflows are that you have which are not available in Logs Explorer yet